mix.exs

```diff
@@ -63,7 +63,8 @@ defmodule Kaffe.Mixfile do
     [
       extras: [
         "README.md": [title: "Overview"],
-        "LICENSE.md": [title: "License"]
+        "LICENSE.md": [title: "License"],
+        "CHANGELOG.md": [title: "Changelog"]
```

Comment:
Nice addition! Are you able to run `mix docs` for this repo? I get an error about an unclosed backtick 🤔